pkg/frontend/kubeactions/kubeactions.go

```diff
@@ -179,13 +178,31 @@ func (ka *kubeactions) CreateOrUpdate(ctx context.Context, oc *api.OpenShiftClus
 		return err
 	}
 
+	objbytes, err := obj.MarshalJSON()
+	if err != nil {
+		return err
+	}
+
 	_, err = dyn.Resource(*gvr).Namespace(namespace).Update(obj, metav1.UpdateOptions{})
 	if !errors.IsNotFound(err) {
 		return err
 	}
 
+	if errors.IsNotFound(err) {
+		createaction = true
+	}
+
 	_, err = dyn.Resource(*gvr).Namespace(namespace).Create(obj, metav1.CreateOptions{})
-	return err
+	if err != nil {
+		return err
+	}
+
+	if createaction {
+		ka.log.Infof("Creating obj: %s", objbytes)
+	} else {
+		ka.log.Infof("Updating obj: %s", objbytes)
```

Comment:
It might be better to log the difference. look at cmp.Diff()

something like : https://github.com/Azure/ARO-RP/blob/aro-operator/pkg/util/cmp/cmp.go#L10

https://github.com/Azure/ARO-RP/blob/aro-operator/pkg/dynamichelper/dynamichelper.go#L253-L259

Reply:
Prior to this we were using `cmp.Diff()` only for tests. `cmp.Diff()` panics a lot if it can not compare values. Not sure it's a good idea to put it into prod code.

Also diff is going to be multi line. It probably going to be not very readable in our logging system.